_IBC/archway-migaloo.json

```diff
@@ -0,0 +1,31 @@
+{
+  "$schema": "../ibc_data.schema.json",
+  "chain_1": {
+    "chain_name": "migaloo",
```

Comment:
chain_1 must be `archway` in this case, not `migaloo` (to follow alphabetical order)

Reply:
@JeremyParish69 Corrected